BUILD

```diff
@@ -27,3 +28,9 @@ sh_test(
         ".bazelversion",
     ],
 )
+
+# gazelle:build_file_name BUILD
+# gazelle:exclude kythe
+# gazelle:exclude third_party
```

Comment:
Can we add more exclusions/patches to remove the spurious warnings gazelle emits?  Things like:

```
org_golang_x_tools: gazelle: /home/schroederc/.cache/bazel/_bazel_schroederc/ddace01a1ad38df359eb57115be59fad/external/org_golang_x_tools/cmd/fiximports/testdata/src/old.com/bad/bad.go: error reading go file: /home/schroederc/.cache/bazel/_bazel_schroederc/ddace01a1ad38df359eb57115be59fad/external/org_golang_x_tools/cmd/fiximports/testdata/src/old.com/bad/bad.go:2:43: expected 'package', found 'EOF'
```

Reply:
Not here, but I've added it to the appropriate rule.